src/api/route-services/experiment.js

```diff
@@ -46,6 +46,30 @@ class ExperimentService {
     return data;
   }
 
+  async getListOfExperiments(experimentIds) {
+    const dynamodb = createDynamoDbInstance();
+
+    const params = {
+      RequestItems: {
+        [this.experimentsTableName]: {
+          Keys: [...experimentIds].map((experimentId) => convertToDynamoDbRecord({ experimentId })),
```

Comment:
Is creating a copy of experimentIds necessary here? I mean the `[...experimentIds]`

Reply:
Hmm... yea, they don't have to be copied. Fixed.